### PR TITLE
Add option to show top bar with logo and username

### PR DESCRIPTION
**Pull request by alepev**
Issue description: https://trello.com/c/pUDfcooi
Relates to https://github.com/thehyve/opentargets-genetics-automation/pull/29

### CODE REVIEW

File: apps/genetics/src/config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const config = {
   platformUrl: window.configPlatformUrl
     ? window.configPlatformUrl.replace(/\/$/, '')
     : 'https://platform.opentargets.org',
+  showTopBar: window.configShowTopBar ?? true,
```
[the remark this answers is not in the document]
Comment (alepev) [reply apps/genetics/src/config.js]: fixed

**Review by RobinM-code — APPROVED** [review]
Nice work!

**Review by louwenjjr — APPROVED** [review]
Looks good to me. Nice job!

File: apps/platform/src/components/NavBar.jsx [hunk 2]
```diff
@@ -90,58 +129,65 @@ const NavBar = ({
...
-          <Button component={ReactRouterLink} to="/" color="inherit">
-            <OpenTargetsTitle name={name} />
-          </Button>
-        )}
-        <div className={classes.flex} />
-        {search ? search : null}
+      {/* push the content down so it isn't hidden behind the logo bar */}
+      {config.showTopBar &&
+          <div id="placeholder-div" style={{ height: '50px', width: '100%' }} />}
+      <Toolbar variant="dense" className={classNames(classes.spaceBetween)}>
```
Comment (fedde-s) [inline apps/platform/src/components/NavBar.jsx:135]: Do these lines (apart from the one above that inserts a div) have to change? I don't think they had to before, and it looks like a lot more of a hassle to maintain when EBI releases new versions, especially if they shouldn't change for the platform version.

File: apps/genetics/src/components/NavBar/NavBar.jsx [hunk 3]
```diff
@@ -94,58 +124,63 @@ const NavBar = ({
...
-
-        {docs ? (
-          <MenuExternalLink classes={classes} href={docs}>
-            Docs
-          </MenuExternalLink>
-        ) : null}
+      {/* push the content down so it isn't hidden behind the logo bar */}
+      {config.showTopBar &&
+          <div id="placeholder-div" style={{ height: '50px', width: '100%' }} />}
+      <Toolbar variant="dense" className={classes.spaceBetween}>
```
Comment (fedde-s) [inline apps/genetics/src/components/NavBar/NavBar.jsx:130]: Do these lines (apart from the one above that inserts a div) have to change? I don't think they had to before, and it looks like a lot more of a hassle to maintain when EBI releases new versions, especially if they shouldn't change for the platform version.

File: apps/platform/src/components/NavBar.jsx (hunk 2 above)
Comment (fedde-s) [reply apps/platform/src/components/NavBar.jsx:135]: As we discussed in the office last night, these extra classes and wrappers were likely mostly due to differences in layout between the newer ot-ui-apps code on which the top bar was previously built and the v0.2.6 state used by OTG release 22.10. There might be cleaner solution to be found if we were to spend more time on it, but if the next release of OTG will build on a newer state of ot-ui-apps anyway (the widgets in EBI's deployment already look more modern, even though it's still on data release 22.10), my maintainability comments are irrelevant as long as we can find back the developments that we do need to keep in later releases. :+1:

File: apps/genetics/src/components/NavBar/NavBar.jsx (hunk 3 above)
Comment (fedde-s) [reply apps/genetics/src/components/NavBar/NavBar.jsx:130]: See [discussion under the other occurrence of this code](https://github.com/thehyve/ot-ui-apps/pull/2#discussion_r1171458452).

File: apps/platform/src/components/NavBar.jsx (hunk 2 above)
Comment (alepev) [reply apps/platform/src/components/NavBar.jsx:135]: Agreed. Shall I capture this in a Trello ticket (backlog for now) to remember what we should do when the time comes?

Comment (louwenjjr) [reply apps/platform/src/components/NavBar.jsx:135]: Neh don't think so. We will see upon a new release.

**Pull request merged**